test/javascripts/acceptance/canned-replies-test.js.es6

```diff
@@ -62,8 +62,8 @@ acceptance("Canned Replies", {
           {
             id: "04697870e02acfef3c2130dab92fe6d8",
             title: "Using variables",
-            excerpt: "%{user}",
-            content: "%{user}"
+            excerpt: "Hi %{reply_to_username,fallback:there}, regards %{my_username}.",
```

Comment:
Can you also add a test that test all the variables at once to ensure they all work properly?

Reply:
Sure, I'll add more tests next week.